### DISCUSSION TITLE
Curly bracket {} usualy higlighted in vs code but not showing when a php closing tag break it

### DISCUSSION THREAD

**Discussion by tybeey** [community/community#49400] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have an issue with Curly brackets{}. So basicaly when I click on the opening curly bracket it is higlighting the closing one.

its working fine here:
![02](https://user-images.githubusercontent.com/67636221/223385361-ccf6e458-b996-4087-aa5b-47147f997862.png)

It was working perfectly fine until yesterday but somehow if a php tag for example "?>" break it then both turn red and not higligting the closing Curly bracket anymore.

As you see its higlighting the opening and closing php tags  on curly bracket click instead of the one i want to be higlighted:
![01](https://user-images.githubusercontent.com/67636221/223385407-b3dcde1c-9b07-4fe5-80f5-3095643a9b2f.png)

I've got two laptops and on my main it isnt working but my other one its working perfectly fine. any idea ? or do you know how can i save and import the other visual code settings over to this one ? I tried to sync them and also re-installed the windows last night but its still not working and the same error showing...

Any idea to resolve this issue ?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @tybeey ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
